ts_src/payments/index.ts

```diff
@@ -17,11 +19,14 @@ export interface Payment {
   pubkeys?: Buffer[];
   input?: Buffer;
   signatures?: Buffer[];
+  internalPubkey?: Buffer;
   pubkey?: Buffer;
   signature?: Buffer;
   address?: string;
   hash?: Buffer;
   redeem?: Payment;
+  redeemVersion?: number;
+  scriptTree?: Taptree;
```

Comment:
Should `scriptTree` be renamed to `tapTree`  (for `psbt` consistency sake)? Please vote:
 - 👍 (yes)
 - 👎 (no)
- 😕  (doesn't matter)

Reply:
😕 (doesn't matter)